README.md

```diff
@@ -9,7 +9,7 @@
 
 Try it out:
 ```bash
-docker run -it -p 127.0.0.1:8443:8443 -v "${PWD}:/home/coder/project" codercom/code-server --allow-http --no-auth
+docker run -it -p 127.0.0.1:8443:8443 -v "${PWD}/code-server:/home/coder" codercom/code-server --allow-http --no-auth
```

Comment:
I am not too sure about this change. Are we assuming that there will be `code-server` available under `${PWD}/`?

Reply:
It will be created if it doesn't exist - I think it's better than polluting the $PWD with the rnd, .local and cache as it does now.